x-pack/plugins/index_management/server/routes/api/enrich_policies/register_list_route.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IScopedClusterClient } from '@kbn/core/server';
+import { RouteDependencies } from '../../../types';
+import { addBasePath } from '..';
+import { enrichPoliciesActions } from '../../../lib/enrich_policies';
+
+export function registerListRoute({ router, lib: { handleEsError } }: RouteDependencies) {
+  router.get(
+    { path: addBasePath('/enrich_policies'), validate: false },
```

Comment:
I was thinking that we should consider adding new routes under `/internal/index_management` base path. This would clearly indicate other plugins to now use the route directly and rely on the provided api service instead. It's less effort to do it before the route is published than change it later, wdyt?